gravatar/src/main/java/com/gravatar/services/IdentityService.kt

```diff
@@ -0,0 +1,93 @@
+package com.gravatar.services
+
+import com.gravatar.logger.Logger
+import com.gravatar.restapi.models.Identity
+import com.gravatar.restapi.models.SelectAvatar
+import kotlinx.coroutines.withContext
+import okhttp3.OkHttpClient
+import com.gravatar.di.container.GravatarSdkContainer.Companion.instance as GravatarSdkDI
+
+/**
+ * Service for managing Gravatar identities.
+ */
+public class IdentityService(private val okHttpClient: OkHttpClient? = null, oauthToken: String) {
```

Comment:
❓ Why do we pass the auth token in the constructor? It's also a param in each function.

Reply:
I forgot to remove it. Good catch.